### DISCUSSION TITLE
What is a good first issue?

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#718] [General]
https://discordapp.com/channels/714698561081704529/714830398315823204/723212534420733973

**Comment by filiptronicek** [comment]
I think it's something you should be able to at 3AM 😂

**Comment by bdougie** [comment]
These are good answers, my addition to the discussion are good-first-issues must have enough details to be worked on by anyone without a lot of experience in the project.

## Must have
- A good description of the bug. All questions filled out. 
- Reproduction steps (also reproducible)
- Solution or recommendation on how to fix it.
